plugins/org.eclipse.n4js/xsemantics-gen/org/eclipse/n4js/xsemantics/InternalTypeSystem.java

```diff
@@ -4,12 +4,10 @@
  * are made available under the terms of the Eclipse Public License v1.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-v10.html
- *
+ * 
```

Comment:
This space is introduced by Xsemantics (The patch that @jpilgrim added). In order to get rid of this space, we would have to adjust Xsemantics slightly.